### PR TITLE
dialog qo'shildi

### PR DESCRIPTION
**Pull request by BaxtiyorNurmatov**

### CODE REVIEW

**Comment by kiberravshanbek** [comment]
salom

**Comment by F1rdavs23** [comment]
;kjlkjlkjljllnlnljlnjhkjlj;ijklniknljilnjbkhijubhiuijbkhjiubkhojilnkjbhjilkbhinjkhb

**Review by Nomonjon0124 — APPROVED** [review]

**Comment by AbduvaliyevSamandar** [comment]
Salom hammaga

**Comment by SatimovDeveloper** [comment]
ok

**Comment by F1rdavs23** [comment]
hello world

**Comment by Abduazim15** [comment]
Follow bosila

**Comment by Nomonjon0124** [comment]
chat bo'pketiptiyu

**Comment by F1rdavs23** [comment]
hello world

**Comment by F1rdavs23** [comment]
samandar qani?

**Review by SatimovDeveloper — APPROVED** [review]

**Review by AbduvaliyevSamandar — COMMENTED** [review]
sssss

**Review by OlloshukurAnvarov — CHANGES_REQUESTED** [review]
Changing

**Review by kiberravshanbek — APPROVED** [review]
salom

**Review by Nurali020314 — COMMENTED** [review]
image

**Comment by Erkinovmuhammad** [comment]
klsfwknek

File: app/src/main/java/uz/gita/lesson29git/MainActivity.kt [hunk 1]
```diff
@@ -2,10 +2,15 @@ package uz.gita.lesson29git
...
+import androidx.appcompat.app.AlertDialog
 
 class MainActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+        AlertDialog.Builder(this)
+            .setTitle("Title")
+            .setMessage("message")
+            .show()
```
Comment (OlloshukurAnvarov) [inline app/src/main/java/uz/gita/lesson29git/MainActivity.kt:14]: hgvghv

**Review by OlloshukurAnvarov — CHANGES_REQUESTED** [review]

File: app/src/main/java/uz/gita/lesson29git/MainActivity.kt [hunk 2]
```diff
@@ -2,10 +2,15 @@ package uz.gita.lesson29git
 
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import androidx.appcompat.app.AlertDialog
```
Comment (kiberravshanbek) [inline app/src/main/java/uz/gita/lesson29git/MainActivity.kt:5]: codeni yaxshilash kerak

File: app/src/main/java/uz/gita/lesson29git/MainActivity.kt [hunk 3]
```diff
@@ -2,10 +2,15 @@ package uz.gita.lesson29git
...
 
 class MainActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+        AlertDialog.Builder(this)
```
Comment (Nurali020314) [inline app/src/main/java/uz/gita/lesson29git/MainActivity.kt:11]: Ajoyib

File: app/src/main/java/uz/gita/lesson29git/MainActivity.kt (hunk 2 above)
Comment (muxtorovnodir) [inline app/src/main/java/uz/gita/lesson29git/MainActivity.kt:5]: Alert dialogmi bu?

**Review by muxtorovnodir — COMMENTED** [review]
Alert

**Review by kiberravshanbek — CHANGES_REQUESTED** [review]

File: app/src/main/java/uz/gita/lesson29git/MainActivity.kt (hunk 2 above)
Comment (Nurali020314) [reply app/src/main/java/uz/gita/lesson29git/MainActivity.kt:5]: .

**Comment by muxtorovnodir** [comment]
Tayyor chat ekanku bu!

**Review by SatimovDeveloper — CHANGES_REQUESTED** [review]
sdsdsds

**Pull request merged**